bindings.cpp

```diff
@@ -1815,7 +1819,10 @@ bool DeRestPluginPrivate::checkSensorBindingsForAttributeReporting(Sensor *senso
             {
                 val = sensor->getZclValue(*i, 0x0035); // battery alarm mask
             }
-            else if (sensor->modelId() == QLatin1String("Motion Sensor-A") || sensor->modelId() == QLatin1String("SMSZB-120"))
+            else if (sensor->modelId() == QLatin1String("Motion Sensor-A") ||
+                     sensor->modelId() == QLatin1String("SMSZB-120") ||
+                     sensor->modelId() == QLatin1String("WISZB-120") ||
+                     sensor->modelId().startsWith(QLatin1String("ZHMS101"))
```

Comment:
Compilation fails due missing parentheses, it should be: `sensor->modelId().startsWith(QLatin1String("ZHMS101")))`

Reply:
Strange my local compile goes through. Nevermind, fixed it, thanks!